client/src/main/java/org/jboss/fuse/mvnd/client/DaemonConnector.java

```diff
@@ -144,7 +144,7 @@ private void handleStopEvents(Collection<DaemonInfo> idleDaemons, Collection<Dae
                     stopEvent.getUid(), stopEvent.getTimestamp(), stopEvent.getReason());
         }
 
-        LOGGER.debug(generate(busyDaemons.size(), idleDaemons.size(), recentStopEvents.size()));
+        System.out.println(generate(busyDaemons.size(), idleDaemons.size(), recentStopEvents.size()));
```

Comment:
Why System.out and not LOGGER.info() ?

Reply:
Mainly to avoid the `[INFO] ` prefix on the line.